.github/scripts/amd/package_triton_wheel.sh

```diff
@@ -0,0 +1,95 @@
+set -ex
+
+# Check if ROCM_VERSION argument is provided
+if [[ -z "$1" ]]; then
+    echo "ROCM_VERSION argument not provided setting to default."
+    ROCM_VERSION="6.0"
```

Comment:
> Suggest not leaving default value in the script and instead providing it as input always

Are we okay to grab the ROCm version from the image in the same we handled in our fork?
https://github.com/ROCm/pytorch/pull/1369/

Reply:
@jithunnair-amd for now I've handled this by extract the ROCM_VERSION out of the image inside `package_triton_wheel.sh` by finding the rocm_version.h file.

We can potentially develop a future PR to directly input ROCM_VERSION from the yml script, but I wanted to avoid trouble linting issues here that have appeared before.